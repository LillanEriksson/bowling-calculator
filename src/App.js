/* eslint-disable linebreak-style */
import React from 'react';
import { Provider } from 'react-redux';
import { combineReducers, configureStore } from '@reduxjs/toolkit';

import { bowling } from './reducers/bowling';

const reducer = combineReducers({ bowling: bowling.reducer });

const store = configureStore({ reducer });

export const App = () => {
	return (
		<Provider store={store}>
			<div>Test</div>
		</Provider>
	);
};
