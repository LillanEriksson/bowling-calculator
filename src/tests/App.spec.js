import React from 'react';
import { render } from '@testing-library/react';
import { App } from '../App';

describe('App', () => {
	it('should take snapshot', () => {
		const { asFragment } = render(<App />);
		expect(asFragment(<App />)).toMatchSnapshot();
	});
});
