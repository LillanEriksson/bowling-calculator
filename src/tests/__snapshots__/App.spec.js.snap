// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-gKsewC fTzkMn"
  >
    <div>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        0
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        1
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        2
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        3
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        4
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        5
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        6
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        7
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        8
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        9
      </button>
      <button
        class="sc-jSgupP koksVy"
        type="button"
      >
        10
      </button>
    </div>
    <div
      class="sc-iBPRYJ edZgPn"
    >
      <div
        class="sc-eCssSg cYwXVz"
      >
        <h3>
          0/300
        </h3>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
